src/features/home/components/Map/Map.tsx

```diff
@@ -0,0 +1,144 @@
+import classNames from 'classnames/bind';
+import { GeoJSONSource } from 'mapbox-gl';
+import { useRef, useState, useEffect } from 'react';
+import ReactMapGL, { Layer, MapRef, Source, useMap } from 'react-map-gl';
+
+import 'mapbox-gl/dist/mapbox-gl.css';
+import { Experience } from '@/shared/types/Experience';
+
+import Slider from '../Slider';
+
+import {
+  clusterLayer,
+  unclusteredPointLayer,
+  clusterCountBackgroundLayer,
+  clusterCountLayer,
+} from './layers';
+import styles from './Map.module.scss';
+
+const cx = classNames.bind(styles);
+
+const mapboxAccessToken = process.env.NEXT_PUBLIC_MAPBOX_TOKEN?.replace(
+  / /g,
+  ''
+);
+
+type MapProps = {
+  records: Experience[];
+};
+
+const MapImage = ({ id, src }: { id: string; src: string }) => {
+  const { current: map } = useMap();
+
+  if (map && !map.hasImage(id)) {
+    map.loadImage(src, (error, image) => {
+      if (error) throw error;
+      if (!map.hasImage(id) && image) {
+        map.addImage(id, image);
+      }
+    });
+  }
+
+  return null;
+};
+
+const Map = ({ records }: MapProps) => {
+  const mapRef = useRef<MapRef>(null);
+  const [selectedItems, setSelectedItems] = useState<Experience[]>([]);
+  const [data, setData] = useState({
+    type: 'FeatureCollection',
+    features: records?.map((record: Experience) => ({
+      type: 'Feature',
+      properties: record,
+      geometry: {
+        type: 'Point',
+        coordinates: [Number(record.lat), Number(record.lng)],
+      },
+    })),
+  });
+  const mapStyleUrl = 'mapbox://styles/jinho1011/cl5faqrml00dv15qvknh8tres';
```

Comment:
오.. 이 링크는 뭔가요?? 상단에 상수로 빼면 좋을 것 같습니다
```suggestion
  const MAP_STYLE_URL = 'mapbox://styles/jinho1011/cl5faqrml00dv15qvknh8tres';
```